tardis/plasma/standard_plasmas.py

```diff
@@ -73,7 +73,10 @@ def assemble_plasma(config, model, atom_data=None):
             raise ValueError('No atom_data option found in the configuration.')
 
         logger.info('Reading Atomic Data from %s', atom_data_fname)
-        atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+        try:
+            atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+        except TypeError:
+            logger.warn('Use new configuration for the atomic database')
```

Comment:
@laudb you need to reraise the exception because it doesn't work with the old dataset. In this reraised exception you should suggest that the error likely occured due to the use of an old-format atomic database. Then also point to where to download a new one.

Reply:
@wkerzendorf okay, I think I understand what you mean, will make changes to the exception and modify the logged message